report_qweb_encrypt/static/src/js/report/action_manager_report.esm.js

```diff
@@ -0,0 +1,79 @@
+/** @odoo-module **/
+
+import {Dialog} from "@web/core/dialog/dialog";
+import {download} from "@web/core/network/download";
+import {registry} from "@web/core/registry";
+const {Component} = owl;
+
+async function download_function(action, options, env) {
+    const type = action.report_type === "qweb-pdf" ? "pdf" : action.report_type;
+    let url = `/report/${type}/${action.report_name}`;
+    const actionContext = action.context || {};
+    const userContext = Object.assign(env.services.user.context, {
+        encrypt_password: action.context.encrypt_password,
```

Comment:
did you mean encrypted?

Reply:
No, `encrypt_password` is the context key that points to the password that is fetched in the backend to encrypt the pdf - here:

https://github.com/OCA/reporting-engine/blob/e9536926133fc3f0a79bcdd226b40cdd56059675/report_qweb_encrypt/models/ir_actions_report.py#L49

so the name was intentional.